.github/workflows/ci.yml

```diff
@@ -199,8 +199,7 @@ jobs:
           CI: true
 
       - name: Build
-        run: yarn build
-        working-directory: ${{ matrix.target }}
+        run: node scripts/smoke/index.js
```

Comment:
Any reason to do a script and not have a specific `test:smoke` command like this?

```bash
turbo run build --scope=docs --scope=www --scope="@example/*"
```

Reply:
ah good call, I like this